pkg/config/nodetreemodel/getter.go

```diff
@@ -112,14 +112,48 @@ func (c *ntmConfig) inferTypeFromDefault(key string, value interface{}) (interfa
 	return deepcopy.Copy(value), nil
 }
 
+func (c *ntmConfig) nodeAtPath(key string) Node {
```

Comment:
The only caller of this is `getNodeValue`? It should get inlined, so we can reduce the number of methods in the implementation that do nearly the same thing.